runtimeconfig/manager_test.go

```diff
@@ -143,14 +288,14 @@ func TestManager_ListenerWithDefaultLimits(t *testing.T) {
 	require.NoError(t, services.StartAndAwaitRunning(context.Background(), overridesManager))
 
 	// check if the metrics is set to the config map value before
-	assert.NoError(t, testutil.GatherAndCompare(reg, strings.NewReader(fmt.Sprintf(`
+	assert.NoError(t, testutil.GatherAndCompare(reg, strings.NewReader(`
 					# HELP runtime_config_hash Hash of the currently active runtime config file.
 					# TYPE runtime_config_hash gauge
-					runtime_config_hash{sha256="%s"} 1
+					runtime_config_hash{sha256="fa368abb9506cf551faee8f92705fa5d5e1e3cf17ad43bb01f69ff64bac7c301"} 1
```

Comment:
When we marshal the yaml file in runtimeconfig manager we add a trailing new line at the end, I was unable to replicate that formatting here in the unit test, so I went with the set hash from the manager.

Reply:
when updating `config` to this:

```
	config := []byte(`overrides:
    user1:
        limit2: 150
`)
```

then the previous version which calculates `sha256.Sum256` in the test works